app/settings.py

```diff
@@ -47,3 +47,6 @@ def get_key(key_name):
 DISPLAY_DATETIME_FORMAT = '%A %d %B %Y at %H:%M'
 DATETIME_FORMAT = "%Y-%m-%dT%H:%M:%SZ"
 EUROPE_LONDON = pytz.timezone("Europe/London")
+
+# JWT configurations
+EQ_JWT_LEEWAY = 120
```

Comment:
Add _IN_SECONDS to name so it's obvious it's seconds and explain what this actually affects in the comment...